index.js

```diff
@@ -0,0 +1,17 @@
+const technologies = ['HTML', 'CSS', 'Javascript', 'MongoDB', 'NodeJS']
+const labels = document.querySelector('.labels')
+
+function label(tech) {
+  console.log(tech)
+  tech.map(name => {
+    console.log(name)
```

Comment:
when you test your code and it works, you should remove the console.log to avoid messing with your user's console 👍